alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -277,19 +304,21 @@ transTx ::
     HasField "body" tx (Core.TxBody era),
     HasField "wits" tx (TxWitness era)
   ) =>
+  EpochInfo Identity ->
+  SystemStart ->
   UTxO era ->
   tx ->
   P.TxInfo
-transTx utxo tx =
+txInfo ei sysS utxo tx =
   P.TxInfo
-    { P.txInfoInputs = mapMaybe (transTxIn utxo) (Set.toList (inputs' tbody)),
-      P.txInfoInputsFees = mapMaybe (transTxIn utxo) (Set.toList (inputs_fee' tbody)),
-      P.txInfoOutputs = mapMaybe transTxOut (foldr (:) [] outs),
+    { P.txInfoInputs = mapMaybe (txInfoIn utxo) (Set.toList (inputs' tbody)),
+      P.txInfoInputsFees = mapMaybe (txInfoIn utxo) (Set.toList (inputs_fee' tbody)),
+      P.txInfoOutputs = mapMaybe txInfoOut (foldr (:) [] outs),
       P.txInfoFee = (transValue (inject @(Mary.Value (Crypto era)) fee)),
       P.txInfoForge = (transValue forge),
       P.txInfoDCert = (foldr (\c ans -> transDCert c : ans) [] (certs' tbody)),
       P.txInfoWdrl = Map.toList (transWdrl (wdrls' tbody)),
-      P.txInfoValidRange = (transVI interval),
+      P.txInfoValidRange = (transVI ei sysS interval), -- TODO replace with transVITime when possible
```

Comment:
can we not use it now?

Reply:
I will ask @michaelpj if accepting `UTCTime` is ready on the Plutus side

Reply:
Sorry, we should have done this week but we didn't. I made https://jira.iohk.io/browse/SCP-2214, I think Jann will look at it next week.